mysequencer.py

```diff
@@ -0,0 +1,17 @@
+from bgp import ia
+from bgp import Sequencer
+from bgp.modules.terms import TocPageDetectorModule, PageTypeProcessor, CopyrightPageDetectorModule
+
+
+PageTypeDetectionSequencer = Sequencer({
+    "pagetypes": PageTypeProcessor(modules={
+        "toc_page": TocPageDetectorModule()
+    })
+})
+
+
+book = ia.get_item("9780262517638OpenAccess") 
+
+results = PageTypeDetectionSequencer.sequence(book).results
+
+print(results)
```

Comment:
This is okay for testing but we'll want to delete this file before we move out of draft

Reply:
Yeah absolutely. I made this file for my convenience during development. When we decide to merge, I will make a commit to delete this file.